### PR TITLE
BCN Mihai&Miriam

### PR DESCRIPTION
**Pull request by cosmihai**
@MiriamAparicio

### CODE REVIEW

**Comment by MiriamAparicio** [comment]
`SortedList.prototype.max     = function() {
  return Math.max.apply(null,this.items);
};

SortedList.prototype.min     = function() {
  return Math.min.apply(null,this.items);
};`

Other options to get max and min from and array of numbers

File: starter-code/index.js [hunk 1]
```diff
@@ -1,10 +1,52 @@
...
+};
+
+SortedList.prototype.add = function(item) {
+
+  this.item.push(item);
+  this.item.sort(function (a,b) {
+    return a-b;
+  });
+  this.length = this.item.length;
+}
```
Comment (tawebbcn) [inline starter-code/index.js]: :ok_hand:

File: starter-code/index.js [hunk 2]
```diff
@@ -1,10 +1,52 @@
...
+
+SortedList.prototype.get = function(pos) {
+  return this.item[pos-1];
```
Comment (tawebbcn) [inline starter-code/index.js]: :heavy_check_mark:

File: starter-code/index.js [hunk 3]
```diff
@@ -1,10 +1,52 @@
...
+}
+
+
+SortedList.prototype.max = function() {
+  let max = this.item[0];
+  this.item.forEach(function (element) {
```
Comment (tawebbcn) [inline starter-code/index.js]: same as ```Math.max()```

File: starter-code/index.js [hunk 4]
```diff
@@ -1,10 +1,52 @@
...
+  this.item.forEach(function(element) {
+    if (element < min) {
+      min = element;
+    }
+  });
+  return min;
+}
+
+SortedList.prototype.average = function() {
+  return this.sum()/this.length
```
Comment (tawebbcn) [inline starter-code/index.js]: Niiiiiice!

File: starter-code/index.js [hunk 5]
```diff
@@ -1,10 +1,52 @@
...
+}
+ 
+
+SortedList.prototype.sum = function() {
+  return this.item.reduce(function (acc, curr) {
+    return acc+=curr;
+  }, 0)
```
Comment (tawebbcn) [inline starter-code/index.js]: :ok_hand:

**Pull request closed without merging**